packages/shared/components/popups/LedgerNotConnected.svelte

```diff
@@ -1,17 +1,18 @@
 <script>
     import { Button, Icon, Text } from 'shared/components'
     import { closePopup } from 'shared/lib/popup'
+    import { stopPollingLedgerStatus } from 'shared/lib/ledger'
 
     export let message
     export let handleClose
     export let locale
 
     function handleCancelClick() {
+        stopPollingLedgerStatus()
         if ('function' === typeof handleClose) {
```

Comment:
You could also do this like this :) 
`typeof handleClose === 'function' && handleClose()`